### DISCUSSION TITLE
How do I include client side routes?

### DISCUSSION THREAD

**Discussion by agamm** [react-static/react-static#1443] [General]
Hi,
I have a landing page that should be rendered statically (node) and a `/app` path that I want to render in the browser (client-side, basically a create-react app).

How can I achieve this kind of flow with react-static (with minimal changes to the `/app` project?

**Comment by warloque** [comment]
Why not just serve your react-static landing separately from your CRA app? 
Ex.: site.com for RS landing, and site.com/app for, you know, the app ;-)

**Reply by agamm** [reply]
> Why not just serve your react-static landing separately from your CRA app?
> Ex.: site.com for RS landing, and site.com/app for, you know, the app ;-)

That is the solution I thought about (actually using a subdomain app.site.com) but it seemed like it would be nice to be able to serve it from the statically generated output and also prefetch the page like react-static does. Maybe I'm asking too much...
